app/src/main/java/org/jellyfin/androidtv/util/profile/MediaCodecCapabilitiesTest.kt

```diff
@@ -165,4 +165,30 @@ class MediaCodecCapabilitiesTest {
 
 		return false
 	}
+
+	fun getMaxResolution(): Pair<Int, Int> {
+		var maxWidth = 0
+		var maxHeight = 0
+
+		for (info in mediaCodecList.codecInfos) {
+			if (info.isEncoder) continue
+
+			try {
+				val capabilities = info.getCapabilitiesForType(MediaFormat.MIMETYPE_VIDEO_AVC)
```

Comment:
I'd move the media type to a parameter so the function is re-usable for different codecs